new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/BuildApplication.java

```diff
@@ -75,233 +52,176 @@
 
   private static String image;
   private static boolean isUseSecret;
-  private static final String APPLICATIONS_MOUNT_PATH = "/application";
-  private static final String SCRIPTS_MOUNT_PATH = "/buildScripts";
+  private static final String APPLICATIONS_PATH = "/u01/application";
   private static final String BUILD_SCRIPT = "build_application.sh";
   private static final Path BUILD_SCRIPT_SOURCE_PATH = Paths.get(RESOURCE_DIR, "bash-scripts", BUILD_SCRIPT);
 
-  private static final ConditionFactory withStandardRetryPolicy
-      = with().pollDelay(2, SECONDS)
-          .and().with().pollInterval(10, SECONDS)
-          .atMost(5, MINUTES).await();
-
   /**
    * Build application.
    *
-   * @param application path of the application source folder
-   * @param parameters system properties for ant
-   * @param targets ant targets to call
-   * @param namespace name of the namespace to use for pvc
+   * <p>The appSrcPath, your application source directory is zipped up and copied to a WebLogic server pod for building.
+   * The application is built under /u01/application inside the pod. If your archives are placed under
+   * /u01/application/build after building, use <b>build</b> as the archiveDistDir param value. This method copies the
+   * folder /u01/application/<b>archiveDistDir</b> to local file system under
+   * WORK_DIR/<b>your_application_name</b>/u01/application/<b>archiveDistDir</b>
```

Comment:
Can we just copy the ear/war file to WORK_DIR/your_application_name? /u01 is generally in the pod for our weblogic servers. So, it is a bit confusing.

Reply:
The Kubernetes java api Copy.copyDirectoryFromPod(...) which actually returns the entire absolute path of the directory inside the pod and hence the /u01/application/...

I can get rid of the /u01/application and can form a path like WORK_DIR/applicationName/archiveDistDir ,  since we cannot put a restriction on what output directories the applications build system will use, the archiveDistDir  will be required parameter.

Reply:
lgtm